bcda/cclf/testutils/cclfUtils.go

```diff
@@ -61,13 +60,14 @@ func ImportCCLFPackage(acoSize, environment string) (err error) {
 	}
 
 	dateStr := fmt.Sprintf("%s.D%s", time.Now().Format("06"), time.Now().Format("060102"))
-	for _, file := range files {
-		filename := fmt.Sprintf("T.A%s.ACO.%sY%s.T%s", acoIDNum, file.Name(), dateStr, time.Now().Format("1504059"))
-		archiveName := fmt.Sprintf("T.BCD.ACO.%sY%s.T%s%s", file.Name(), dateStr, acoIDNum, "000")
-		err = zipTo(fmt.Sprintf("%s/%s", sourcedir, file.Name()), filename, fmt.Sprintf("%s/%s", DestDir, archiveName))
-		if err != nil {
-			return err
-		}
+	for i, file := range files {
+		timeStr := time.Now().Add(time.Minute * time.Duration(i-1)).Format("1504059")
+			archiveName := fmt.Sprintf("T.BCD.A%s.ZCY%s.T%s", acoIDNum, dateStr, timeStr)
+			filename := fmt.Sprintf("T.BCD.A%s.%sY%s.T%s", acoIDNum, file.Name(), dateStr, "0000000")
+			err = zipTo(fmt.Sprintf("%s/%s", sourcedir, file.Name()), filename, fmt.Sprintf("%s/%s", DestDir, archiveName))
```

Comment:
I ran your branch locally to try to understand why the tests are running successfully in Travis but failing in `dev`.  It looks like these test files are configured incorrectly.  This code creates 3 zips - one for cclf0, one for cclf8, and one for cclf9.  We can't use `zipTo()` here since that creates a new zip file for every file passed to it... we need to instead create an archive and add files to it.  An example can be found here: https://golangcode.com/create-zip-files-in-go/